plotly/matplotlylib/mplexporter/exporter.py

```diff
@@ -162,8 +162,8 @@ def crawl_legend(self, ax, legend):
                 if isinstance(child, matplotlib.patches.Patch):
                     self.draw_patch(ax, child, force_trans=ax.transAxes)
                 elif isinstance(child, matplotlib.text.Text):
-                    if not (child is legend.get_children()[-1]
-                            and child.get_text() == 'None'):
+                    if (child is not legend.get_children()[-1]
+                            and child.get_text() != 'None'):
```

Comment:
@theengineear 

This fixes #384 ..
I can made it as a different PR, or we can use this PR to kill two birds with a single bullet.. 😸